doc/reference/guidelines.rst

```diff
@@ -869,32 +874,31 @@ the form ``_('literal string')`` and nothing else:
     # and the following will of course not work as already explained:
     error = _("Product %s is out of stock!" % product.name)
 
-    # bad: field values are automatically translated by the framework
-    # This is useless and will not work the way you think:
-    error = _("Product %s is not available!") % _(product.name)
-    # and the following will of course not work as already explained:
-    error = _("Product %s is not available!" % product.name)
-
```

Comment:
Why did you remove this?  Those examples are still bad!

Reply:
it's actually duplicated, the exact same examples are just above